docs/content/en/docs/reference/packagespec/metallb/v0.13.5.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "v0.13.5"
+linkTitle: "v0.13.5"
+weight: 20
+description: >
+---
+
+{{% pageinfo %}}
+[FRRouting](https://frrouting.org/) is currently not supported for MetalLB.
+{{% /pageinfo %}}
+
+### Configuring MetalLB in EKS Anywhere package spec
+
+{{% pageinfo %}}
+Starting at v0.13.5, keys within each config section starts with lowercase. For example:
+```
+L2Advertisements:
+    - IPAddressPools:
+        - default
+```
+Becomes:
+```
+L2Advertisements:
+    - ipAddressPools:
+        - default
+```
+Sections remain capitalized as they represent CRDs.
+{{% /pageinfo %}}
+
+#### Example
+```
+apiVersion: packages.eks.amazonaws.com/v1alpha1
+kind: Package
+metadata:
+  name: mylb
+  namespace: eksa-packages
+spec:
+  packageName: metallb
+  targetNamespace: metallb-system
+  config: |
+    ipAddressPools:
+      - name: default
+        addresses:
+          - 10.220.0.93/32
+          - 10.220.0.94/32
+          - 10.220.0.95/32
+      - name: bgp
+        addresses:
+          - 10.220.0.97-10.220.0.99
+    L2Advertisements:
+      - ipAddressPools:
+          - default
+    BGPAdvertisements:
+      - ipAddressPools:
+          - bgp 
+        autoAssign: false
+    BGPPeers:
+      - myASN: 123
+        peerASN: 55001
+        peerAddress: 1.2.3.4
+        keepaliveTime: 30s
+```
+
+| Parameter | Description | Default | Required |
+|---|---|---|---|
+| IPAddressPools[] | A list of **IPAddressPool**. | None | False |
```

Comment:
```suggestion
| ipAddressPools[] | A list of **ipAddressPools**. | None | False |
```
Not sure if I understood what you were saying. Should it be "A list of IP address ranges" or something like that?